main/core/Library/Mailing/TransportFactory.php

```diff
@@ -12,72 +12,73 @@
 namespace Claroline\CoreBundle\Library\Mailing;
 
 use Claroline\CoreBundle\Library\Configuration\PlatformConfigurationHandler;
+use Monolog\Logger;
```

Comment:
Better use high-level abstractions.

```suggestion
use Psr\Log\LoggerInterface;
```